docs/scripts/api/buildApi.ts

```diff
@@ -144,7 +145,7 @@ async function run() {
   });
 }
 
-yargs
+yargs(hideBin(process.argv))
```

Comment:
Do you now what this does ?

Reply:
Yes, [it does the same](https://yargs.js.org/docs/) what `process.argv.slice(2)` would do, but this usage is in their [getting started guide](https://github.com/yargs/yargs?tab=readme-ov-file#simple-example)... 🤔